backend/src/sdk.cpp

```diff
@@ -210,6 +242,99 @@ restinio::request_handling_status_t SDK::handleTxSDKCall(const restinio::request
     return req->create_response().set_body(absl::StrJoin(outData, ",")).done();
 }
 
+void SDK::handleSetStationState(const nlohmann::json json)
```

Comment:
This gets called by the StreamdDeck plugin to set the state of the Rx/Tx/etc. The incoming value can either be "toggle" or true/false for each of the different states.

Reply:
I see you have basic validation but since this is coming from outside, we can't trust the user, please add validation for the whole json packet

Reply:
Nevermind, re-reading it seems to be good, but give it a double check

Reply:
All the logic here is just a refactor from existing code you had that wasn't in a standalone function.